Makefile

```diff
@@ -1,5 +1,6 @@
 export GIT_SHA1=$(git rev-parse --short HEAD)
 export IMAGE_NAME=graylog
+export IMAGE_NAME_FORWARDER=graylog-forwarder
```

Comment:
does the image name align with the product name? we should be careful here as in the past we had some quirks with that topic.